### PR TITLE
amp-ad type=dotandads implementation

### PR DESCRIPTION
**Pull request by dotandads**
Provides support for amp-ad type=dotandads

### CODE REVIEW

**Comment by dotandads** [comment]
I think there are some problems with lint. It's returning "TypeError: Cannot read property 'visitClass' of undefined" in Travis Build

**Comment by cramforce** [comment]
Do you see the same error when running `gulp lint` locally?

File: examples/dotandads.amp.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <head>
+        <meta charset="utf-8">
+        <title>Amp test</title>
+        <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+        <link rel='canonical' href='dotandads.amp.html'>
+        <script async src="https://cdn.ampproject.org/v0.js"></script>
+        <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+    </head>
+    <body>
+        <amp-ad width=980 height=250
```
Comment (cramforce) [inline examples/dotandads.amp.html]: Sorry, I missed this before: Please merge this into `ads.amp.html`. We're trying to keep the count of examples somewhat limited.

**Comment by cramforce** [comment]
Running lint on a clean branch to check whats up https://travis-ci.org/ampproject/amphtml/builds/106271563

**Comment by dotandads** [comment]
Yes, I have the same error locally

File: examples/dotandads.amp.html (hunk 1 above)
Comment (dotandads) [inline examples/dotandads.amp.html]: ok, I'll merge them

**Comment by cramforce** [comment]
Confirmed it is not your issue https://travis-ci.org/ampproject/amphtml/builds/106271563

Please address the example-comment above and we can merge this. Thanks!

**Comment by dotandads** [comment]
Ok, I've fixed the example and squashed the commits ^^

Thank you

**Comment by dotandads** [comment]
Do we have to make some fixes for the conflicts that the check is showing?

**Comment by cramforce** [comment]
@dotandads You do need to rebase from master, so we can merge your PR. After that the build should also be green.

**Comment by dotandads** [comment]
@cramforce ok, everything looks green now

File: ads/dotandads.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    data-sp='sn-u' 
+    data-cid="11" 
+    data-mpo="ampTest" 
+    data-mpt="amp-amp-all-all">
+</amp-ad>
+```
+
+## Configuration
+
+For semantics of configuration, please see ad network documentation.
```
Comment (cramforce) [inline ads/dotandads.md]: Maybe in a follow up send a PR with a link, so people know where to look.

**Comment by cramforce** [comment]
LGTM. See comment above for a follow up. Merging.

**Pull request merged**